### PR TITLE
Doc: add download link and script in orca installation.

### PR DESCRIPTION
**Pull request by lalalapotter**
## Description

This PR is to add download link and scripts in orca installation doc.

### CODE REVIEW

File: docs/readthedocs/source/doc/Orca/Overview/install.md [hunk 1]
```diff
@@ -44,6 +44,8 @@ conda activate py37
 ---
 ## Install BigDL Orca
 
+In this section, you can use `pip install` to install BigDL Orca package, which is the most recommended way to install.
```
Comment (hkvision) [inline docs/readthedocs/source/doc/Orca/Overview/install.md:47]: This section demonstrates how to install BigDL Orca via pip, which is the most recommended way.

File: docs/readthedocs/source/doc/Orca/Overview/install.md [hunk 2]
```diff
@@ -121,3 +123,31 @@ pip uninstall bigdl-orca-spark3 bigdl-dllib-spark3 bigdl-tf bigdl-math bigdl-cor
 ```
 
 __Note__: If necessary, you need to manually uninstall `pyspark` and other [dependencies](https://github.com/intel-analytics/BigDL/tree/main/python/requirements/orca) introduced by Orca.
+
+---
+## Download BigDL Orca
+
+If you need BigDL release packages, you can find the download links of LTS versions and nightly build version for both Spark 2.4.6 and Spark 3.1.2 in the form below. Besides, you can use the [script](https://github.com/intel-analytics/BigDL/blob/main/docker/bigdl-k8s/download-bigdl.sh) to download by specifying the environment variables `BIGDL_VERSION` and `SPARK_VERSION`.
```
Comment (hkvision) [inline docs/readthedocs/source/doc/Orca/Overview/install.md:130]: You can also download the BigDL package via the download links below.

File: docs/readthedocs/source/doc/Orca/Overview/install.md (hunk 1 above)
Comment (lalalapotter) [reply docs/readthedocs/source/doc/Orca/Overview/install.md]: Fixed.

File: docs/readthedocs/source/doc/Orca/Overview/install.md [hunk 3]
```diff
@@ -121,3 +123,31 @@ pip uninstall bigdl-orca-spark3 bigdl-dllib-spark3 bigdl-tf bigdl-math bigdl-cor
...
+    <tr>
+        <td></td>
+        <td>BigDL 2.0.0</td>
+        <td>BigDL 2.1.0</td>
+        <td>BigDL 2.2.0-SNAPSHOT nightly build</td>
+    </tr>
+    <tr>
+       <td>Spark 2.4.6 </td>
+       <td><a href="https://repo1.maven.org/maven2/com/intel/analytics/bigdl/bigdl-assembly-spark_2.4.6/2.0.0/bigdl-assembly-spark_2.4.6-2.0.0.zip">download</a></td>
+       <td><a href="https://repo1.maven.org/maven2/com/intel/analytics/bigdl/bigdl-assembly-spark_2.4.6/2.1.0/">download</a></td>
```
Comment (hkvision) [inline docs/readthedocs/source/doc/Orca/Overview/install.md:144]: this link is wrong, should be the zip

File: docs/readthedocs/source/doc/Orca/Overview/install.md [hunk 4]
```diff
@@ -121,3 +123,31 @@ pip uninstall bigdl-orca-spark3 bigdl-dllib-spark3 bigdl-tf bigdl-math bigdl-cor
...
+       <td><a href="https://oss.sonatype.org/content/repositories/snapshots/com/intel/analytics/bigdl/bigdl-assembly-spark_2.4.6/2.2.0-SNAPSHOT/">download</a></td>
+    </tr>
+    <tr>
+       <td>Spark 3.1.2 </td>
+       <td><a href="https://repo1.maven.org/maven2/com/intel/analytics/bigdl/bigdl-assembly-spark_3.1.2/2.0.0/bigdl-assembly-spark_3.1.2-2.0.0.zip">download</a></td>
+       <td><a href="https://repo1.maven.org/maven2/com/intel/analytics/bigdl/bigdl-assembly-spark_3.1.2/2.1.0/">download</a></td>
```
Comment (hkvision) [inline docs/readthedocs/source/doc/Orca/Overview/install.md:150]: same as above

File: docs/readthedocs/source/doc/Orca/Overview/install.md [hunk 5]
```diff
@@ -121,3 +123,31 @@ pip uninstall bigdl-orca-spark3 bigdl-dllib-spark3 bigdl-tf bigdl-math bigdl-cor
...
+        <td>BigDL 2.2.0-SNAPSHOT nightly build</td>
+    </tr>
+    <tr>
+       <td>Spark 2.4.6 </td>
+       <td><a href="https://repo1.maven.org/maven2/com/intel/analytics/bigdl/bigdl-assembly-spark_2.4.6/2.0.0/bigdl-assembly-spark_2.4.6-2.0.0.zip">download</a></td>
+       <td><a href="https://repo1.maven.org/maven2/com/intel/analytics/bigdl/bigdl-assembly-spark_2.4.6/2.1.0/">download</a></td>
+       <td><a href="https://oss.sonatype.org/content/repositories/snapshots/com/intel/analytics/bigdl/bigdl-assembly-spark_2.4.6/2.2.0-SNAPSHOT/">download</a></td>
+    </tr>
+    <tr>
+       <td>Spark 3.1.2 </td>
```
Comment (hkvision) [inline docs/readthedocs/source/doc/Orca/Overview/install.md:148]: should now be spark 3.1.3?

File: docs/readthedocs/source/doc/Orca/Overview/install.md [hunk 6]
```diff
@@ -121,3 +123,31 @@ pip uninstall bigdl-orca-spark3 bigdl-dllib-spark3 bigdl-tf bigdl-math bigdl-cor
...
+If you need BigDL release packages, you can find the download links of LTS versions and nightly build version for both Spark 2.4.6 and Spark 3.1.2 in the form below. Besides, you can use the [script](https://github.com/intel-analytics/BigDL/blob/main/docker/bigdl-k8s/download-bigdl.sh) to download by specifying the environment variables `BIGDL_VERSION` and `SPARK_VERSION`.
+
+<table border="1"
+cellpadding="10"
+>
+    <tr>
+        <td></td>
+        <td>BigDL 2.0.0</td>
+        <td>BigDL 2.1.0</td>
+        <td>BigDL 2.2.0-SNAPSHOT nightly build</td>
```
Comment (hkvision) [inline docs/readthedocs/source/doc/Orca/Overview/install.md:139]: reverse the order? 2.0.0 is the least important version.

File: docs/readthedocs/source/doc/Orca/Tutorial/yarn.md [hunk 7]
```diff
@@ -309,7 +309,7 @@ On the __Client Node__:
 export SPARK_HOME=/path/to/spark # the folder path where you extract the Spark package
 export SPARK_VERSION="downloaded spark version"
 ```
-2. Download and unzip a BigDL assembly package from [BigDL Assembly Spark 2.4.6](https://repo1.maven.org/maven2/com/intel/analytics/bigdl/bigdl-assembly-spark_2.4.6/2.1.0/bigdl-assembly-spark_2.4.6-2.1.0-fat-jars.zip) or [BigDL Assembly Spark 3.1.2](https://repo1.maven.org/maven2/com/intel/analytics/bigdl/bigdl-assembly-spark_3.1.2/2.1.0/bigdl-assembly-spark_3.1.2-2.1.0-fat-jars.zip) (according to your Spark version), then setup the environment variables `${BIGDL_HOME}` and `${BIGDL_VERSION}`.
+2. Download and unzip a BigDL assembly package from [Download BigDL Orca](../Overview/install.html#download-bigdl-orca), then setup the environment variables `${BIGDL_HOME}` and `${BIGDL_VERSION}`.
```
Comment (hkvision) [inline docs/readthedocs/source/doc/Orca/Tutorial/yarn.md:312]: Refer to [here] to download and unzip...

Comment (hkvision) [reply docs/readthedocs/source/doc/Orca/Tutorial/yarn.md:312]: Add `Make sure the Spark version of your downloaded BigDL matches your downloaded Spark.`

**Review by hkvision — APPROVED** [review]

**Pull request merged**